address/address.go

```diff
@@ -118,14 +119,18 @@ type Tap struct {
 	// assetGen is the receiving asset's genesis metadata which directly
 	// maps to its unique ID within the Taproot Asset protocol.
 	assetGen asset.Genesis
+
+	// ProofCourierAddr is the address of the proof courier that will be
+	// used to distribute related proofs for this address.
+	ProofCourierAddr *url.URL
```

Comment:
Any reason to use a pointer over actual value here?

Reply:
No. I will fix this in a future PR. I don't think this field should be a pointer because it is a mandatory field.